lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteRepositoryIndexSearcher.java

```diff
@@ -113,14 +113,19 @@ public RemoteRepositoryIndexSearcher(MavenLemminxExtension lemminxMavenPlugin, P
 				return indexDownloadJobs.get(res).thenApply(theVoid -> context);
 			}
 			final IndexingContext context = initializeContext(repositoryUrl);
-			indexingContexts.put(repositoryUrl, context);
-			CompletableFuture<IndexingContext> future;
-			try {
-				future = updateIndex(context).thenApply(theVoid -> context);
-				indexDownloadJobs.put(context, future);
-				return future;
-			} catch (ComponentLookupException e) {
-				return CompletableFuture.failedFuture(e);
+			if (context != null) {
+				indexingContexts.put(repositoryUrl, context);
+				CompletableFuture<IndexingContext> future;
+				try {
+					future = updateIndex(context).thenApply(theVoid -> context);
+					indexDownloadJobs.put(context, future);
+					return future;
+				} catch (ComponentLookupException e) {
+					System.out.println("getIndexingContext(): logging out exeption: " + e.getLocalizedMessage());
```

Comment:
Please use a LOGGER here.

Reply:
This trace was added for debug purpose so, I'd better remove this line.